backend/src/main/java/hu/flowacademy/konyhatunder/service/RecipeService.java

```diff
@@ -65,9 +65,6 @@ public void validate(EmptyRecipe emptyRecipe){
         if(emptyRecipe.getLevel() == null)
             throw new ValidationException("Nem jó level");
 
-        if(emptyRecipe.getCategoryList() == null || emptyRecipe.getCategoryList().size() == 0)
-            throw new ValidationException("A kategóriák megadása kötelező");
-
```

Comment:
Attól még serveren is lehet validálni.

Reply:
Véletlen került bele ez a validálás korábban, elkerülte a figyelmünket hogy kategóriát nem kötelező a feltöltőnek megadnia, ezértért töröltük.

Reply:
Tesztelés volt a célja , hogy a backend hiba üzenetek áttjönnek - e / működnek -e és az üzenet átadás megy .